### PR TITLE
Load typekit

### PR DESCRIPTION
**Pull request by jessicabarbalho**

### CODE REVIEW

**Comment by jessicabarbalho** [comment]
@julianalucena take a look here please ;)

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -7,3 +7,10 @@
 //= require placeholder-polyfill.min
 //= require bootstrap-redu
 //= require olark
+
+$.getScript("http://use.typekit.com/lpo4rgu.js", function () {
```
Comment (julianalucena) [inline app/assets/javascripts/application.js]: Where is the definition of `getScript`?

Comment (jessicabarbalho) [inline app/assets/javascripts/application.js]: It's a method from [jQuery](http://api.jquery.com/jQuery.getScript/)

Comment (julianalucena) [inline app/assets/javascripts/application.js]: Cool! Did you see the `cache` parameter?
Just to check, did you simulate the occasion when the JS is not available? Just to see if the page appears as we expect. :smiley:

Comment (jessicabarbalho) [inline app/assets/javascripts/application.js]: I will take a look :eyes: /o/

**Comment by jessicabarbalho** [comment]
@julianalucena I made some modifications

File: app/assets/javascripts/application.js [hunk 2]
```diff
@@ -19,3 +19,19 @@ $(function() {
...
+    url: url
+  };
+
+  return $.ajax(options);
+};
+
+$.cachedScript("http://use.typekit.com/lpo4rgu.js").done(function() {
+    try {
+        Typekit.load();
+        $(".typekit-badge").css("left", "0");
```
Comment (julianalucena) [inline app/assets/javascripts/application.js]: Minnor identation error.

File: app/assets/javascripts/application.js (hunk 1 above)
Comment (julianalucena) [inline app/assets/javascripts/application.js]: Did you simulate it? :snowman:

Comment (jessicabarbalho) [inline app/assets/javascripts/application.js]: Yes

**Pull request merged**